packages/data-context/src/DataContext.ts

```diff
@@ -79,6 +79,7 @@ export class DataContext {
   private _config: Omit<DataContextConfig, 'modeOptions'>
   private _modeOptions: Readonly<Partial<AllModeOptions>>
   private _coreData: CoreDataShape
+  private _versionsDataSource: VersionsDataSource | undefined
```

Comment:
What is this property used for ?

Reply:
Good catch [`f907132` (#20107)](https://github.com/cypress-io/cypress/pull/20107/commits/f907132d50fb9c8db6e2679dcd9291fa4e2d4cb3)